[2021-04-21 20:20:40,672] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-19T22:40:00+00:00 [queued]>
[2021-04-21 20:20:40,694] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-19T22:40:00+00:00 [queued]>
[2021-04-21 20:20:40,695] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:20:40,696] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-21 20:20:40,697] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:20:40,718] {taskinstance.py:1063} INFO - Executing <Task(DockerOperator): docker_op_tester> on 2021-04-19T22:40:00+00:00
[2021-04-21 20:20:40,722] {standard_task_runner.py:52} INFO - Started process 1532 to run task
[2021-04-21 20:20:40,727] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'docker_op_tester', '2021-04-19T22:40:00+00:00', '--job-id', '594', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmpstmm1gp8', '--error-file', '/tmp/tmp19xh07i5']
[2021-04-21 20:20:40,733] {standard_task_runner.py:77} INFO - Job 594: Subtask docker_op_tester
[2021-04-21 20:20:40,821] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.docker_op_tester 2021-04-19T22:40:00+00:00 [running]> on host 5af2b1cab26e
[2021-04-21 20:20:40,900] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=docker_op_tester
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T22:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T22:40:00+00:00
[2021-04-21 20:20:40,925] {docker.py:224} INFO - Starting docker container from image centos:latest
[2021-04-21 20:21:11,571] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=docker_op_tester, execution_date=20210419T224000, start_date=20210421T202040, end_date=20210421T202111
[2021-04-21 20:21:11,600] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-21 20:21:11,616] {local_task_job.py:146} INFO - Task exited with return code 0
